packages/desktop/components/popups/send/SendNftForm.svelte

```diff
@@ -0,0 +1,175 @@
+<script lang="ts">
+    import { get } from 'svelte/store'
+    import { Error } from '@ui'
+    import { closePopup, openPopup, PopupId } from '@auxiliary/popup'
+    import { localize } from '@core/i18n'
+    import { selectedAccount } from '@core/account/stores'
+    import { Output } from '@iota/sdk/out/types'
+    import {
+        getDefaultTransactionOptions,
+        getOutputParameters,
+        NewNftTransactionDetails,
+        newTransactionDetails,
+        NewTransactionType,
+        setNewTransactionDetails,
+    } from '@core/wallet'
+    import { Button, NetworkInput, NftInput, OptionalInput, RecipientInput } from 'shared/components'
+    import features from '@features/features'
+    import { activeProfile } from '@core/profile'
+    import { prepareOutput } from '@core/account'
+    import { handleError } from '@core/error/handlers'
+    import { OptionalInputType } from '../send'
+    import { MAX_METADATA_BYTES, MAX_TAG_BYTES } from '@core/utils'
+
+    export let validateInputs: (
+        inputValidations?: any,
+        optionalInputValidations?: {
+            validate?: (promise: Promise<unknown>) => Promise<void>
+            value: string
+            optionalInputType: OptionalInputType
+            byteLimit: number
+            errorMessage: string
+        }[]
+    ) => Promise<boolean> = () => Promise.resolve(true)
+    export let isTransferring: boolean = false
+
+    let preparedOutput: Output
+    let isPreparingOutput = false
+
+    // Inputs
+    let nftInput: NftInput
```

Comment:
1.  Owned NFTs not displayed in the nftInput dropdown
![image](https://github.com/iotaledger/firefly/assets/52735957/5d70df3b-88c1-43d3-889a-39c0d0c2d708)

2.  Sending NFT from Collection Send button opens Send Form with Token tab active
![image](https://github.com/iotaledger/firefly/assets/52735957/c0878c2a-1d08-4895-a725-68c13f96b823)

3. switching to NFT tab nft not selected in nftInput
![image](https://github.com/iotaledger/firefly/assets/52735957/2e9b40f4-427c-465e-bdbd-ec6d0803e7b1)